### PR TITLE
[IND-387] add script to parse block and print json

### PR DESCRIPTION
**Pull request by dydxwill**
### Changelist
add script to parse block and return json

### Test Plan
ran on devbox

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: indexer/services/scripts/src/helpers/block-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  StatefulOrderEventV1,
+  SubaccountUpdateEventV1,
+  TransferEventV1,
+  UpdateClobPairEventV1,
+  UpdatePerpetualEventV1,
+} from '@dydxprotocol-indexer/v4-protos';
+
+import { AnnotatedIndexerTendermintEvent } from './types';
+
+export enum DydxIndexerSubtypes {
```
Comment (Christopher-Li) [inline indexer/services/scripts/src/helpers/block-helpers.ts:19]: nit: move to types

File: indexer/services/scripts/src/print-block.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    message: `Connecting to kafka brokers: ${config.KAFKA_BROKER_URLS}`,
+  });
+  await startKafka(height);
+  logger.info({
+    at: 'index#start',
+    message: `Successfully connected to kafka brokers: ${config.KAFKA_BROKER_URLS}`,
+  });
+}
+
+const args = yargs.options({
```
Comment (Christopher-Li) [inline indexer/services/scripts/src/print-block.ts:160]: nit: only if it's easy to do, can we also add support to just parse a block at a particular offset

**Review by vincentwschau — APPROVED** [review]

File: indexer/services/scripts/src/helpers/block-helpers.ts (hunk 1 above)
Comment (will530) [reply indexer/services/scripts/src/helpers/block-helpers.ts:19]: done

File: indexer/services/scripts/src/print-block.ts (hunk 2 above)
Comment (will530) [reply indexer/services/scripts/src/print-block.ts:160]: I don't think this is needed. I believe all debug logs in Ender that print out offset should also print out block height, so we can just use the block height.

File: indexer/services/scripts/src/helpers/block-helpers.ts (hunk 1 above)
Comment (dydxwill) [reply indexer/services/scripts/src/helpers/block-helpers.ts]: done

**Review by Christopher-Li — APPROVED** [review]

**Pull request merged**